docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Projection/VectorWhiten.cs

```diff
@@ -58,5 +58,38 @@ public static void Example()
             // 0.454  0.523  0.593  0.664  1.886 -0.757 -0.687 -0.022  0.176  0.310
             // 0.863  0.938  1.016  1.093 -1.326 -0.096 -0.019  0.189  0.330  0.483
         }
+
+        private class SampleVectorOfNumbersData
+        {
+            [VectorType(10)]
+            public float[] Features { get; set; }
+        }
+
+        /// <summary>
+        /// Returns a few rows of the infertility dataset.
+        /// </summary>
+        private static IEnumerable<SampleVectorOfNumbersData> GetVectorOfNumbersData()
+        {
+            var data = new List<SampleVectorOfNumbersData>();
+            data.Add(new SampleVectorOfNumbersData { Features = new float[10] { 0, 1, 2, 3, 4, 5, 6, 7, 8, 9 } });
+            data.Add(new SampleVectorOfNumbersData { Features = new float[10] { 1, 2, 3, 4, 5, 6, 7, 8, 9, 0 } });
+            data.Add(new SampleVectorOfNumbersData
+            {
+                Features = new float[10] { 2, 3, 4, 5, 6, 7, 8, 9, 0, 1 }
+            });
+            data.Add(new SampleVectorOfNumbersData
+            {
+                Features = new float[10] { 3, 4, 5, 6, 7, 8, 9, 0, 1, 2, }
+            });
+            data.Add(new SampleVectorOfNumbersData
+            {
+                Features = new float[10] { 5, 6, 7, 8, 9, 0, 1, 2, 3, 4 }
+            });
+            data.Add(new SampleVectorOfNumbersData
+            {
+                Features = new float[10] { 6, 7, 8, 9, 0, 1, 2, 3, 4, 5 }
+            });
```

Comment:
ditto.

Reply:
similar change. please create the data inline with initializers, similar to NgramExtraction.

---
In reply to: [278763353](https://github.com/dotnet/machinelearning/pull/3594#discussion_r278763353) [](ancestors = 278763353)